packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -254,7 +255,7 @@ const EAPostsItem = ({classes, ...props}: EAPostsListProps) => {
             {hasAudio && <ForumIcon icon="VolumeUp" />}
           </div>
           <div className={classes.tag}>
-            {primaryTag && <FooterTag tag={primaryTag} smallText />}
+            {showTag && primaryTag && <FooterTag tag={primaryTag} smallText />}
```

Comment:
`primaryTag` is also returned by the same call to `usePostsItem` so I think it would probably be cleaner just have `usePostsItem` set `primaryTag` to `null` when `hideTag` is set, then we don't need to introduce this extra `showTag` variable.